log_subscriber/compose.yml

```
@@ -0,0 +1,32 @@
+    depends_on:
+      - redis
+    networks:
+      - reearth-flow-net
+    environment:
+      - PUBSUB_EMULATOR_HOST=pubsub:8085
+      - STORAGE_EMULATOR_HOST=http://gcs:4443
+      - FLOW_LOG_SUBSCRIBER_PROJECT_ID=local-project
+      - FLOW_LOG_SUBSCRIBER_SUBSCRIPTION_ID=flow-log-stream-topic-sub
+      - FLOW_LOG_SUBSCRIBER_REDIS_ADDR=redis:6379
+      - FLOW_LOG_SUBSCRIBER_REDIS_PASSWORD=
+      - FLOW_LOG_SUBSCRIBER_GCS_BUCKET_NAME=reearth-flow-oss-bucket
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)